README.md

```diff
@@ -55,3 +57,7 @@ Run the tests with:
 ```node
 $ npm test
 ```
+
+## Sentry
+To configure sentry, create a new project at https://sentry.wnyc.org/sentry/. 
+The creation of that project will automatically generate a DSN. It will look something like this: `https://<KEY>@sentry.wnyc.org/<PROJECT_ID>`. Create an environment variable in circleci called `<ENV>_SENTRY_DSN` and set it to the value sentry provided in project creation. 
```

Comment:
This is a little confusing, as the name of the environment variable isn't as important as the key name on the passed in options object. Last tweak and then we can release: could you update this line so that it's clear that the user of this library must pass in a value at the `sentryDSN` key, not that they must provide an envvar of some name? i.e. this module should have no knowledge of whatever envvars are set.

Reply:
Fair enough. I'll fix that.

Reply:
thank you!